README.md

```diff
@@ -15,26 +15,24 @@ Welcome to the Athens project! We are a proxy server for the [Go Modules downloa
 |------------------|
-|We hold weekly developer meetings on a Thursday, to join them, watch previous meeting recordings or find more information, please see [the docs](https://docs.gomods.io/contributing/community/developer-meetings/). Absolutely everyone is invited to attend these, suggest topics, and participate!|
+|We hold weekly developer meetings every Thursday. To join them, watch previous meeting recordings or find more information, please see [the docs](https://docs.gomods.io/contributing/community/developer-meetings/). Absolutely everyone is invited to attend these, suggest topics, and participate!|
 </br>
 
 See our documentation site [https://docs.gomods.io](https://docs.gomods.io) for more details on the project.
 
 # Project Status
 
-Project Athens is in alpha. Things might change, so we recommend that you don't run it for production workloads. We have organizations that are testing it internally, and there is an experimental public proxy server running.
+Project Athens is in beta. It's being tested and used used on teams and has an experimental deployment for public use.
 
-We encourage you to [test it out](https://docs.gomods.io/install/) and [contribute](#contributing) when you can!
+We encourage you to [try it out](https://docs.gomods.io/install/), let us know what you think, and [contribute](#contributing) when you can!
 
 # More Details Please!
 
-Although the project is alpha, here's where we're going:
-
 The proxy implements the [Go modules download protocol](https://docs.gomods.io/intro/protocol/).
 
-There is currently an experimental public proxy, and we have plans to host a more stable public proxy with more guarantees. We also have a community of folks who are testing Athens inside their organizations, as an "internal proxy." In either deployment, users set their `GOPROXY` environment variable to point to the Athens proxy of their choice. At that point, `go get`, `go build`, and `go build`s will use the proxy to download dependencies as necessary.
+There is currently an experimental public proxy, and we have plans to host a more stable public proxy with more guarantees. We also have a community of folks using Athens inside their organizations as an internal proxy. In either deployment, developers set their `GOPROXY` environment variable to point to the Athens proxy of their choice. At that point, `go get`, `go build`, and `go build`s will use the proxy to download dependencies as necessary.
 
-Athens proxies are highly configurable, so they can work for lots of different deployments. For example, public proxies can store code in cloud databases and CDNs, while internal "enterprise" deployments can use disk-based (i.e. NFS) storage.
+Athens proxies are highly configurable, so they can work for lots of different deployments. For example, proxies support a wide variety of storage drivers like cloud blob storage systems, cloud CDNs, shared disk, and even in-memory.
```

Comment:
> cloud CDNs

Could you explain what do you mean by that?

Reply:
Sure, I'll write it up in the docs